ckanext/depositar_theme/fanstatic/scss/_bootstrap-variables.scss

```diff
@@ -0,0 +1,12 @@
+$primary: #000;
+$link-decoration: none;
+$link-hover-decoration: none;
+$badge-color: #fff;
```

Comment:
```suggestion
```
It's [white](https://github.com/ckan/ckan/blob/ckan-2.10.7/ckan/public/base/scss/_bootstrap-variables.scss#L1222) by default.